test/parallel/test-util-inspect.js

```diff
@@ -474,6 +474,7 @@ test_lines({
 
 // test boxed primitives output the correct values
 assert.equal(util.inspect(new String('test')), '[String: \'test\']');
+assert.equal(util.inspect(Object(Symbol('test'))), '[Symbol: Symbol(test)]');
```

Comment:
No, IMO.  It conveys that it's a symbol wrapped in an object; i.e., that `typeof object === 'object'` but that `typeof object.valueOf() === 'symbol'`.